lib/sensu-plugins-gpg/version.rb

```diff
@@ -1,9 +1,15 @@
+# SensuPluginsGpg is the main module
 module SensuPluginsGpg
+  # Version holds details about version number. It consists of
+  # three elements which are major, minor and patch number.
   module Version
+    # Major version number (X.0.0)
     MAJOR = 1
-    MINOR = 0
+    # Minor version number (0.X.0)
+    MINOR = 1
```

Comment:
Please leave version bumps to maintainers, there are a couple reasons:
- there is no guarantee on merge order
- maintainers analyze the change to determine the appropriate version, for example from what I have seen so far this is a major rev as `sensu-plugin` bump is actually breaking and therefore this is wrong.

You can read more about our PR process here: https://github.com/sensu-plugins/community/blob/master/PULL_REQUEST_PROCESS.md